File: javascript/test/conformance-runner.mjs
import { deriveAddress, supportedChains } from "../src/index.js";

const mnemonic = "abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon about";
const result = Object.fromEntries(supportedChains().map(({ id }) => [id, deriveAddress({ source: { mnemonic }, chain: id }).address]));
console.log(`WALLETHD_CONFORMANCE=${JSON.stringify(result)}`);

File: python/tests/conformance_runner.py
import json

from wallet_hd_derivation_kit import derive_address, supported_chains

MNEMONIC = "abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon about"
result = {chain["id"]: derive_address({"mnemonic": MNEMONIC}, chain=chain["id"])["address"] for chain in supported_chains()}
print("WALLETHD_CONFORMANCE=" + json.dumps(result, sort_keys=True, separators=(",", ":")))
